net/shadowsocks-libev/Makefile

```diff
@@ -29,6 +29,7 @@ PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 PKG_USE_MIPS16:=0
 PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=c-ares pcre
```

Comment:
Better use `+=`

Reply:
[Most of the packages here use `PKG_BUILD_DEPENDS :=`](https://github.com/openwrt/packages/search?q=PKG_BUILD_DEPENDS)
Does it matter if the variable is not assigned beforehand?

Reply:
It's a minor nitpick from myself.  If convention has already formed among other parts, forget my comment and let's follow suit.  My intention was for flexibilities like allowing user or build system to have a pre-defined depends.